models/keyword.go

```diff
@@ -18,28 +18,62 @@ type Keyword struct {
 
 	Keyword string `orm:"size(128)"`
 	Url     string `orm:"size(128)"`
+	Status  int    `orm:"default(0)"`
 }
 
+var statusKeyword = map[string]int{"failed": -1, "pending": 0, "completed": 1}
```

Comment:
How about naming this:

```suggestion
var keywordStatuses = map[string]int{"failed": -1, "pending": 0, "completed": 1}
```

Or 

```suggestion
var keywordStatusList = map[string]int{"failed": -1, "pending": 0, "completed": 1}
```